sky/skylet/autostop_lib.py

```diff
@@ -54,13 +54,11 @@ def get_autostop_config() -> AutostopConfig:
 def set_autostop(idle_minutes: int, backend: Optional[str], down: bool) -> None:
     boot_time = psutil.boot_time()
     autostop_config = AutostopConfig(idle_minutes, boot_time, backend, down)
-    prev_autostop_config = get_autostop_config()
     configs.set_config(_AUTOSTOP_CONFIG_KEY, pickle.dumps(autostop_config))
     logger.debug(f'set_autostop(): idle_minutes {idle_minutes}, down {down}.')
-    if (prev_autostop_config.autostop_idle_minutes < 0 or
-            prev_autostop_config.boot_time != psutil.boot_time()):
```

Comment:
Do we handle the previous logic `prev_autostop_config.boot_time != psutil.boot_time()`? Does this handle the case when a cluster is restarted, the idleness timer should reset?

Reply:
Yes, in the current code, we will reset the active time in any cases when the `set_autostop` function is called, including the restarted case.